tensorflow_addons/text/crf.py

```diff
@@ -188,8 +188,9 @@ def crf_log_likelihood(inputs,
 
     # Get the transition matrix if not provided.
     if transition_params is None:
-        transition_params = tf.get_variable("transitions",
-                                            [num_tags, num_tags])
+        initializer = tf.initializers.GlorotUniform()
```

Comment:
Is there any precedent for using GlorotUniform here? Thanks very much for the PR as usual

Reply:
It's the default behavior for `get_variable`.
> https://www.tensorflow.org/api_docs/python/tf/get_variable?hl=en
> If initializer is None (the default), the default initializer passed in the variable scope will be used. If that one is None too, a glorot_uniform_initializer will be used.

Reply:
This was why I used `GlorotUniform`, I could parameterize it with `GlorotUniform` being the default.

Reply:
Hmmm I think parametizer-ing it may be a loverkill. This looks good to me.